src/bin/sage-runtests

```diff
@@ -171,13 +171,18 @@ if __name__ == "__main__":
         pytest_options = []
         if args.verbose:
             pytest_options.append("-v")
-        # #31924: Do not run pytest on individual Python files unless
-        # they match the pytest file pattern.  However, pass names
-        # of directories. We use 'not os.path.isfile(f)' for this so that
-        # we do not silently hide typos.
-        filenames = [f for f in args.filenames
-                     if f.endswith("_test.py") or not os.path.isfile(f)]
-        if filenames or not args.filenames:
+
+        # #35999: no filenames in arguments is equivalent to "src/sage"
+        if not args.filenames:
+            filenames = [os.path.join(os.environ.get('SAGE_SRC'), "sage")]
```

Comment:
```suggestion
            filenames = [os.environ.get('SAGE_SRC')]
```
since otherwise https://github.com/sagemath/sage/blob/develop/src/conftest_inputtest.py is not run

Reply:
OK. Done. Thanks.